### PR TITLE
new chart newrelic-php-daemon

### PR DESCRIPTION
**Pull request by beniamin**
TODO:
- [x]  add anti-affinity by default
- [x]  fix readiness and liveness probes
- [x]  add config for newrelic-daemon

Changed default logfile to `/dev/stdout` to improve logs collection.

### CODE REVIEW

File: charts/newrelic-php-daemon/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
```
Comment (alexbumbacea) [inline charts/newrelic-php-daemon/templates/deployment.yaml:56]: I know it's not a common practice, but I would enable hostname based anti afinity by default.

File: index.yaml [hunk 2]
```diff
@@ -1,9 +1,29 @@
 apiVersion: v1
 entries:
   haproxy:
+  - apiVersion: v2
```
Comment (alexbumbacea) [inline index.yaml:4]: Why do we need haproxy here?

File: charts/newrelic-php-daemon/values.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  repository: newrelic/php-daemon
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "9.13.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```
Comment (alexbumbacea) [inline charts/newrelic-php-daemon/values.yaml:17]: Does this container really require a service account?

Comment (alexbumbacea) [inline charts/newrelic-php-daemon/values.yaml:17]: Does this container really require a service account?

**Review by alexbumbacea — CHANGES_REQUESTED** [review]
This change should not include changes related to haproxy chart

File: charts/newrelic-php-daemon/templates/deployment.yaml (hunk 1 above)
Comment (beniamin) [reply charts/newrelic-php-daemon/templates/deployment.yaml:68]: done

File: charts/newrelic-php-daemon/values.yaml (hunk 3 above)
Comment (beniamin) [reply charts/newrelic-php-daemon/values.yaml:17]: done

**Review by alexbumbacea — APPROVED** [review]

**Pull request merged**